### ISSUE TITLE
Use pymake [or some sort of parallel build] when building xulrunner on windows

### ISSUE THREAD

**Issue by johnmurrayvi** [nightingale-media-player/nightingale-deps#15]
I saw that pymake supported concurrent compilation jobs and does not have deadlock issues on Windows. I'm assuming that problem is why builds aren't using some form of parallel compilation as of now. If not pymake, then maybe there's something else that could be used. Either way, it would definitely be nice to not wait 2+ hours on  xulrunner debug and release builds.

**Comment by mook** [comment]
Yeah, the default mozconfigs should be using -j of some sort on non-Win32
platforms already; pymake or a patched GNU make 4 might allow parallel
builds on Windows too.

**Comment by johnmurrayvi** [comment]
I'm gonna try to get this functional, so I'll assign myself.

**Comment by johnmurrayvi** [comment]
Looks like this may be fixed upstream now with "mozmake":
https://bugzilla.mozilla.org/show_bug.cgi?id=927213

Also, I forgot to add:
https://github.com/nightingale-media-player/nightingale-deps/commit/ac63d386eb0147b173487e1ec174844c0297e738
The -jN flag was being hardcoded to -j1 apparently.

**Comment by johnmurrayvi** [comment]
Changing this to Windows only
